### PR TITLE
[neo][vllm] FP8 Quantization support for partition & Neo

### PR DESCRIPTION
**Pull request by a-ys**

### CODE REVIEW

File: serving/docker/partition/partition.py [hunk 1]
```diff
@@ -276,6 +283,36 @@ def autoawq_quantize(self):
...
         tokenizer.save_pretrained(output_path)
 
+    def autofp8_quantize(self):
+        sys.path.append(PYTHON_CACHE_DIR)
+        from djl_python.properties_manager.hf_properties import HuggingFaceProperties
+
+        output_path = self.properties['option.save_mp_checkpoint_path']
+        properties = remove_option_from_properties(self.properties)
+        hf_configs = HuggingFaceProperties(**properties)
+        tokenizer = init_hf_tokenizer(hf_configs.model_id_or_path, hf_configs)
```
Comment (Lokiiiiii) [inline serving/docker/partition/partition.py]: Seems like 287-293 are duplicated from AWQ path. Please refactor to reduce duplication.

File: serving/docker/partition/sm_neo_quantize.py [hunk 2]
```diff
@@ -109,7 +108,7 @@ def write_properties(self):
                 "option.tensor_parallel_degree"] = user_tensor_parallel_degree
         else:
             logging.info(
-                "User did not passs tensor_parallel_degree. Outputted serving.properties"
+                "User did not passs tensor_parallel_degree. Outputted serving.properties "
```
Comment (Lokiiiiii) [inline serving/docker/partition/sm_neo_quantize.py]: Typo

**Review by Lokiiiiii — COMMENTED** [review]
Based on NeuralMagic's Model Zoo, 
Customers are required to fine-tune all the parameters available in BaseQuantizeConfig.

Can you add support to expose these to the customer ?

**Comment by Lokiiiiii** [comment]
https://huggingface.co/collections/neuralmagic/fp8-llms-for-vllm-666742ed2b78b7ac8df13127

**Comment by a-ys** [comment]
Upstream PR: https://github.com/deepjavalibrary/djl-serving/pull/2272

**Pull request closed without merging**